pkgs/by-name/be/bepass/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, enableGUI ? false # upstream working in progress
+, pkg-config
+, glfw
+, xorg
+, libXcursor
+, libXrandr
+, libXinerama
+, xinput
+, libXi
+, libXxf86vm
+}:
+buildGoModule rec{
+  pname = "bepass";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "bepass-org";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21